src/js/Router.js

```diff
@@ -37,6 +37,10 @@ export default class Router extends String {
      * @return {String}
      */
     toString() {
+        if (!this._params) {
```

Comment:
This is to avoid a failure if calling toString when this._params is not defined, not really publicly exposed but good to have